### DISCUSSION TITLE
Feature request: Status indicator on icon

### DISCUSSION THREAD

**Discussion by imjoyjean** [patternfly/patternfly#5484] [Feature requests]
## Requesting new features, enhancements, or design changes to PatternFly
Describe the use cases and requirements of the new feature or enhancement you are requesting.
- Need an indicator to signal that something is new or needs attention. Could be multiple colors. The default could be blue (info), and yellow could indicate warning, while red could indicate danger. It is a common pattern across the internet currently. Should be familiar with it for those who use apps & smart phones. Example screenshot of mockup:

<img width="254" alt="image" src="https://user-images.githubusercontent.com/57908474/217603528-a5f69d39-5303-4f99-83e8-a0ae25bdfefc.png">

Explain *why* this feature is needed and/or why these requirements cannot be met using existing components.
- There is currently not a simple/clean way to indicate that something is new. I could use status icons, but if the dropdown itself is triggered by an icon, then it would look really complicated with two icons side by side with no hierarchy. 

If this is for specific applications, list them here along with any dates that you need this for.
- Designing for the unified console. Stonesoup may also have use cases for it. 

Link any detailed design documents, rough sketches, wireframes or workflows.
https://docs.google.com/document/d/1b_QXcWk5yDoDXlMRFdhxxzHfWhK-N-_yTOUzFnVtSlQ/edit?usp=sharing

State whether this is a new feature or an enhancement on an existing pattern by labeling it accordingly
- new feature

If this is an enhancement to an existing feature, identify the feature that you want to modify.
- This feature can be applied to a menu item, tab, dropdown, icon, etc.

**Comment by mcarrano** [comment]
@imjoyjean thanks for submitting this request. We will add it to our feature request queue for further consideration after our v5 release.
